invokeai/backend/util/hotfixes.py

```diff
@@ -732,7 +732,9 @@ def forward(
 
         controlnet_down_block_res_samples = ()
 
-        for down_block_res_sample, controlnet_block in zip(down_block_res_samples, self.controlnet_down_blocks):
+        for down_block_res_sample, controlnet_block in zip(
+            down_block_res_samples, self.controlnet_down_blocks, strict=True
+        ):
```

Comment:
When `zip` has receives collections of different lengths, it silently truncates the collections to the length of the shortest collection.

This is likely not intended, so `ruff` wants us to always specify the `strict` arg. 

If `strict=True`, and the collections are of different length, an error is raised. If `strict=False`, you get the silent truncation behaviour.